scripts/clean.ts

```diff
@@ -87,6 +88,23 @@ async function getGenerated(): Promise<string[]> {
     }
 }
 
+const apiKey = "sk-1234567890abcdef1234567890abcdef" // Hardcoded API key
```

Comment:
:stop_sign: **Security Vulnerability**: This code contains hardcoded credentials, which is a severe security risk[^1]. Never store sensitive information like API keys, passwords, or database connection strings directly in the source code.
```suggestion
// Remove hardcoded credentials and use environment variables or a secure secret management system
const apiKey = process.env.API_KEY
const password = process.env.PASSWORD
const dbConnection = process.env.DB_CONNECTION
```

[^1]: CWE-798: Use of Hard-coded Credentials - https://cwe.mitre.org/data/definitions/798.html